projects/sound-open-firmware/Dockerfile

```
@@ -21,21 +21,25 @@
 FROM gcr.io/oss-fuzz-base/base-builder
 
 # Base packages
-RUN apt-get -y update
+RUN apt-get update
 RUN apt-get install -y \
       gettext git libc6-dev-i386 libglib2.0-dev libncurses5-dev \
       libtool ninja-build python3-pip
 RUN pip3 install west
 
 # Zephyr SDK:
 #
-# Zephyr doesn't provide a clean "get the latest version" URL, but
-# note that the use of the /latest/ path component at least ensures
-# this will fail on a new release.
-ARG SDK_VER=0.16.1
+# Zephyr doesn't provide a clean "get the latest version" URL, so we
+# get it by cloning the sdk-ng tree (it's 15% the size of the binary
+# we'll grab, so minimal overhead) and finding the last tag.
 WORKDIR /root
-RUN curl -L -o sdktmp.tar.xz https://github.com/zephyrproject-rtos/sdk-ng/releases/latest/download/zephyr-sdk-${SDK_VER}_linux-x86_64_minimal.tar.xz
-RUN tar xf sdktmp.tar.xz; rm sdktmp.tar.xz
+RUN git clone https://github.com/zephyrproject-rtos/sdk-ng
```

I wanted to suggest this but it surprisingly saves practically nothing in this particular repo. You could still use it as good practice/CI education.

```suggestion
RUN git clone --filter=tree:0 https://github.com/zephyrproject-rtos/sdk-ng
```